source/fundamentals/builders/aggregation-expression-operations.txt

```diff
@@ -138,52 +140,55 @@ the Server manual documentation.
 Arithmetic Operations
 ~~~~~~~~~~~~~~~~~~~~~
 
-You can use arithmetic operations to manipulate numbers. You can perform an
-arithmetic operation on a value of ``MqlInteger`` or ``MqlNumber`` using the
-methods described in this section.
+You can perform an arithmetic operation on a value of type ``MqlInteger`` or
+``MqlNumber`` using the methods described in this section.
 
 .. list-table::
    :header-rows: 1
-   :widths: 40 60
+   :widths: 60 60
```

Comment:
Suggestion:

There isn't a good reason to use a total that's not equal to 100. If you want them to be equal, "50 50" is equivalent.